vesc_driver/src/vesc_interface.cpp

```diff
@@ -161,9 +161,8 @@ void VescInterface::Impl::rxThread()
         error_handler_("Possibly out-of-sync with VESC, read timout in the middle of a frame.");
       }
     }
-
     // Only attempt to read every 10 ms
-    std::this_thread::sleep_for(std::chrono::milliseconds(10));
+//    std::this_thread::sleep_for(std::chrono::milliseconds(5));
```

Comment:
Can you please explain this change?

Reply:
I tried various waiting times but, also introducing the reading of IMU data, any waiting time produced a loss of data.  removing the wait solved the problem.

I'll remove this comments.